docs/_docs/tools/control-script.adoc

```diff
@@ -646,4 +646,136 @@ control.bat --change-tag _<new-tag>_
 ----
 --
 
+== Metric command
 
+Metrics command prints value(s) of the metric or metric registry from the parameter.
+
+If you need to get metric from the specific node, please, set the `--node-id` parameter.
+Without `--node-id` random node will be used.
+
+[tabs]
+--
+tab:Linux[]
+[source,shell,subs="verbatim,quotes"]
+----
+control.sh --metric sys
+----
+tab:Windows[]
+[source,shell,subs="verbatim,quotes"]
+----
+control.bat --metric sys
+----
+--
+
+Example of the metric output:
+[source, text]
+$ ./bin/control.sh --metric sysCurrentThreadCpuTime
+Command [METRIC] started
+Arguments: --metric sys
+--------------------------------------------------------------------------------
+metric                          value
+sys.CurrentThreadCpuTime        17270000
+Command [METRIC] finished with code: 0
+
+
+Example of the metric registry output:
+[source, text]
+$ ./bin/control.sh --metric io.dataregion.default
+Command [METRIC] started
+Arguments: --metric sys
+--------------------------------------------------------------------------------
+metric                          value
+io.dataregion.default.TotalAllocatedSize          0
+io.dataregion.default.LargeEntriesPagesCount      0
+io.dataregion.default.PagesReplaced               0
+io.dataregion.default.PhysicalMemorySize          0
+io.dataregion.default.CheckpointBufferSize        0
+io.dataregion.default.PagesReplaceRate            0
+io.dataregion.default.InitialSize                 268435456
+io.dataregion.default.PagesRead                   0
+io.dataregion.default.AllocationRate              0
+io.dataregion.default.OffHeapSize                 0
+io.dataregion.default.UsedCheckpointBufferSize    0
+io.dataregion.default.MaxSize                     6871947673
+io.dataregion.default.OffheapUsedSize             0
+io.dataregion.default.EmptyDataPages              0
+io.dataregion.default.PagesFillFactor             0.0
+io.dataregion.default.DirtyPages                  0
+io.dataregion.default.TotalThrottlingTime         0
+io.dataregion.default.EvictionRate                0
+io.dataregion.default.PagesWritten                0
+io.dataregion.default.TotalAllocatedPages         0
+io.dataregion.default.PagesReplaceAge             0
+io.dataregion.default.PhysicalMemoryPages         0
+Command [METRIC] finished with code: 0
+
+== System view command
```

Comment:
"== System View Command"

All the first letters must be written in the upper-case.